index.html

```diff
@@ -7658,8 +7658,38 @@ <h5>Presentational Role Inheritance</h5>
 				    <li>If <code>aria-valuemin</code> is missing or not a <a href="#valuetype_number">number</a>, it defaults to 0 (zero). </li>
 				    <li>If <code>aria-valuemax</code> is missing or not a <a href="#valuetype_number">number</a>, it defaults to 100. </li>
 				</ul>
-				<p>Authors MUST set the <pref>aria-valuenow</pref> attribute. If aria-valuenow is missing or has an unexpected value, browsers MAY implement the repair techniques specified in the <a href="#authorErrorDefaultValuesTable">section describing handling author errors in states and properties</a>, which are equivalent to the repair techniques for <code>&lt;input type="[^input/type/range^]"&gt;</code> in HTML.</p>
+				<p>Authors MUST set either the <pref>aria-valuenow</pref> attribute, the <pref>aria-valuetext</pref> attribute, or both. If <code>aria-valuenow</code> is missing or has an unexpected value, browsers MAY implement the repair techniques specified in the <a href="#authorErrorDefaultValuesTable">section describing handling author errors in states and properties</a>, which are equivalent to the repair techniques for <code>&lt;input type="[^input/type/range^]"&gt;</code> in HTML.</p>
 				<p>Elements with the role <code>slider</code> have an implicit <pref>aria-orientation</pref> value of <code>horizontal</code>.</p>
+				<div class="note">
+					<p>It is usually advisable for a slider to express its current value as a parsable <a href="#valuetype_number">number</a> using <pref>aria-valuenow</pref>. However, there are use cases where <pref>aria-valuetext</pref> alone provides a superior user experience.</p>
+					<p>For example, pain can be measured along a range of values in which each value is most helpfully expressed by descriptive text rather than superficially expressed by an unintuitive number.</p>
+				</div>
+				<pre class="example highlight">
+&lt;label&gt;
+  Pain rating:
+  &lt;input id="pain" type="range" min="0" max="10" value="0" aria-valuetext="No pain" /&gt;
```

Comment:
should probably use a fully custom ARIA example.  with the input element, the value is going to still be specified by default.

Reply:
Gah, of course. On it.